how-tos/arxiv-submission.txt

```diff
@@ -0,0 +1,77 @@
+How to make a submission to the ArXiv:
```

Comment:
It would be best to only have one version of the instructions as keeping the two formats in sync will be difficult.  Please, remove `txt` version.